actions/mysql_explain_action_test.go

```diff
@@ -210,5 +210,42 @@ id |select_type |table |partitions |type |possible_keys |key  |key_len |ref  |ro
 			require.NoError(t, err)
 			checkCity(t)
 		})
+
+		t.Run("Stored function", func(t *testing.T) {
+			t.Parallel()
+			// setup
+			func(t *testing.T) {
+				t.Helper()
+				ctx, cancel := context.WithTimeout(context.Background(), time.Second*2)
+				defer cancel()
+				conn, err := db.Conn(ctx)
+				require.NoError(t, err)
+				defer conn.Close() //nolint:errcheck
+
+				_, err = conn.ExecContext(ctx, "DROP FUNCTION IF EXISTS cleanup")
+				require.NoError(t, err)
+				_, err = conn.ExecContext(ctx, `CREATE FUNCTION cleanup() RETURNS char(50) CHARSET latin1
+				DETERMINISTIC
+				BEGIN
+				delete from world.city;
```

Comment:
Let's make a little improvement and create a separate table to manipulate.

Reply:
Done